.github/workflows/build_and_test.yml

```diff
@@ -110,12 +108,17 @@ jobs:
             TEST_CMD="$TEST_CMD --coverage"
           fi
           $TEST_CMD
-    
+
+    - name: Update current time
+      if: ${{ always() && (inputs.coverage == true || github.event_name == 'schedule') }}
+      run: |
+        echo "time_now=$(date +%s%N)" >> $GITHUB_ENV
+
     - name: Archive coverage stats file
       uses: actions/upload-artifact@v3
       if: ${{ always() && (inputs.coverage == true || github.event_name == 'schedule') }}
       with:
-        name: luacov-stats-out-1
+        name: luacov-stats-out-${{ env.time_now }}
```

Comment:
Is there something more unique than time? What about using a combination of the SHA and the name or SHA+time?

Reply:
SHA would default to the last commit on the default branch in case the trigger is [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), I think we could use `github.job` + `github.run_id` instead. That way we could avoid using time altogether and avoid updating it.